playwright/utils/common.ts

```diff
@@ -0,0 +1,30 @@
+import { type Page } from '@playwright/test';
+
+export const acceptCookieConsentBanner = async (page: Page) => {
+  // Check if the cookie banner is present on the page
+  const cookieBanner = page.locator('#ez-cookie-notification');
+  const isBannerVisible = await cookieBanner.isVisible();
+
+  if (isBannerVisible) {
+    // Click the accept button on the cookie banner
+    const acceptButton = page.locator('#ez-cookie-notification__accept');
+
+    await acceptButton.click();
+  }
+};
+
+export const fillInputByLabelWithDelay = async ({
+  page,
+  label,
+  value,
+  timeout = 500,
+}: {
+  page: Page;
+  label: string | RegExp;
+  value: string;
+  timeout?: number;
+}) => {
+  await page.getByLabel(label).pressSequentially(value, { delay: 100 });
+  // eslint-disable-next-line playwright/no-wait-for-timeout
+  await page.waitForTimeout(timeout);
```

Comment:
Adding an extra timeout before pressSequentially seemed to fix things for me.

Reply:
thank you @iamsamgibbs  for your help with that. I’ll add an extra timeout for now, although normally, it shouldn’t be necessary. My assumption is that this issue might be related to the app running in dev mode - e2e tests should be run against the optimized build, with no extra overhead of the dev server, HMR, huge js/css chunks, etc. Playwright, due to its ability to interact with the app very quickly, may be revealing a problem that’s hard to reproduce manually. 
Besides that, I’ve found there could indeed be an issue with at least one particular input field's validation - I was able to reproduce it manually. Here’s the link to the issuehttps://github.com/JoinColony/colonyCDapp/issues/3565

Reply:
but do we actually need timeout after pressSequentially ?

Reply:
ok let me remove it )

Reply:
@Nortsova removed it, is all tests still green for you?